pkg/rancher-desktop/main/diagnostics/limaDarwin.ts

```diff
@@ -0,0 +1,62 @@
+import Electron from 'electron';
+import semver from 'semver';
+
+import { DiagnosticsCategory, DiagnosticsChecker } from './types';
+
+import mainEvents from '@pkg/main/mainEvents';
+import { spawnFile } from '@pkg/utils/childProcess';
+import Logging from '@pkg/utils/logging';
+
+const console = Logging.diagnostics;
+
+let kubernetesMemory = Number.POSITIVE_INFINITY;
+
+mainEvents.on('settings-update', (cfg) => {
+  kubernetesMemory = cfg.kubernetes.memoryInGB;
+});
```

Comment:
This logic doesn't seem to work. After reducing memory to 3GB, the VM was restarted, but the diagnostic remained.

I changed to the Troubleshooting page, and back to Diagnostics, and both the badge on and the diagnostic were still in place.

Pressing the "Re-run" button cleared the diagnostic.